tests/test_bulk_update_or_create.py

```diff
@@ -53,6 +53,9 @@ def test_values(self):
             TestModel, {('test33',): {'int_field': 2}, ('test3',): {'int_field': 2}}, key_fields='name',
             key_is_unique=False))
 
+        self.assertEqual(2, bulk_update_or_create(
```

Comment:
Duplicate: It is not `InputFormat` test. It doesn't check validation issues.
What does it check? I'd rather add it to `TestSimple` class as separate test method.